src/azul/azulclient.py

```diff
@@ -50,15 +49,21 @@ def query(self, prefix=None):
             assert not prefix
             return self._query
 
-    def post_bundle(self, bundle_fqid, indexer_url):
+    def post_bundle(self, indexer_url, notification):
         """
         Send a mock DSS notification to the indexer
         """
-        simulated_event = self._make_notification(bundle_fqid)
-        response = requests.post(indexer_url, json=simulated_event, auth=hmac.prepare())
+        response = requests.post(indexer_url, json=notification, auth=hmac.prepare())
         response.raise_for_status()
         return response.content
 
+    def _make_test_notification(self, bundle_fqid, new_bundle_uuid, test_name, test_uuid):
+        notification = self._make_notification(bundle_fqid)
```

Comment:
Consider 

```suggestion
        return dict(self._make_notification(bundle_fqid), test_bundle_uuid=…, …)
```

Also consider inlining this call as its pretty anemic considering the long signature.

Reply:
I don't think the additional function simplifies anything. The signature is long and brittle and the body is anemic. This doesn't increase readability.